srcpkgs/firefox-esr/template

```diff
@@ -3,47 +3,43 @@
 # THIS PKG MUST BE SYNCHRONIZED WITH "srcpkgs/firefox-esr-i18n".
 #
 pkgname=firefox-esr
-version=52.9.0
-revision=5
-wrksrc="firefox-${version}esr"
+version=60.5.0
+revision=1
+build_helper="rust"
+wrksrc="firefox-${version}"
 short_desc="Mozilla Firefox web browser - Extended Support Release (ESR)"
 maintainer="Eivind Uggedal <eivind@uggedal.com>"
 license="MPL-2.0, GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://www.mozilla.org/firefox/organizations/"
-distfiles="${MOZILLA_SITE}/firefox/releases/${version}esr/source/firefox-${version}esr.source.tar.xz
- https://aur.archlinux.org/cgit/aur.git/plain/ffmpeg4.patch.gz?h=firefox-esr52>ffmpeg4.patch.gz"
-checksum="c01d09658c53c1b3a496e353a24dad03b26b81d3b1d099abc26a06f81c199dd6
- 2a285e0c31968e3fe9b65a585838b46f9342ff0a369e786a729b4c3886617034"
+distfiles="${MOZILLA_SITE}/firefox/releases/${version}esr/source/firefox-${version}esr.source.tar.xz"
+checksum=1a1f69ee87092637f75aef7f3fa588b0eef0b2c8bcc160094a036450c49c4025
 
 lib32disabled=yes
 hostmakedepends="autoconf213 unzip zip pkg-config perl python yasm
- $(vopt_if rust cargo)"
-makedepends="
- libatomic-devel nss-devel gtk+-devel
- sqlite-devel libevent-devel libnotify-devel libvpx-devel
- hunspell-devel libXcomposite-devel libSM-devel libXt-devel
- $(vopt_if alsa alsa-lib-devel) $(vopt_if dbus dbus-glib-devel)
- $(vopt_if gtk3 gtk+3-devel) $(vopt_if pulseaudio pulseaudio-devel)
+ cargo llvm clang rust cbindgen python"
+makedepends="nss-devel libjpeg-turbo-devel gtk+-devel gtk+3-devel icu-devel
+ pixman-devel sqlite-devel libevent-devel libnotify-devel libvpx-devel
+ libXrender-devel libXcomposite-devel libSM-devel libXt-devel rust-std
+ libXdamage-devel rust-std $(vopt_if alsa alsa-lib-devel)
+ $(vopt_if dbus dbus-glib-devel) $(vopt_if pulseaudio pulseaudio-devel)
  $(vopt_if startup_notification startup-notification-devel)
- $(vopt_if xscreensaver libXScrnSaver-devel)"
-depends="nss>=3.27 desktop-file-utils hicolor-icon-theme"
+ $(vopt_if xscreensaver libXScrnSaver-devel) $(vopt_if sndio sndio-devel)"
+depends="nss>=3.37.3 desktop-file-utils hicolor-icon-theme"
 conflicts="firefox>=0"
 
-build_options="alsa dbus gtk3 rust pulseaudio startup_notification xscreensaver"
-build_options_default="alsa dbus gtk3 pulseaudio startup_notification xscreensaver"
-desc_option_rust="Build rust components"
+build_options="alsa dbus gtk3 pulseaudio startup_notification xscreensaver sndio"
+build_options_default="alsa dbus gtk3 pulseaudio startup_notification xscreensaver sndio"
 
-case "$XBPS_TARGET_MACHINE" in
-	i686|x86_64*) build_options_default+=" rust";;
+case $XBPS_TARGET_MACHINE in
+	armv6*)
+		broken="required NEON extensions are not suppored on armv6"
```

Comment:
typo (suppored -> supported)
I wil lfix it in the firefox template next time I toch it

Reply:
Fixed it.